stdlib/_weakrefset.pyi

```diff
@@ -20,15 +20,15 @@ class WeakSet(MutableSet[_T], Generic[_T]):
     def __contains__(self, item: object) -> bool: ...
     def __len__(self) -> int: ...
     def __iter__(self) -> Iterator[_T]: ...
-    def __ior__(self, other: Iterable[_S]) -> WeakSet[_S | _T]: ...
+    def __ior__(self: Self, other: Iterable[_T]) -> Self: ...  # type: ignore[override,misc]
     def difference(self: Self, other: Iterable[_T]) -> Self: ...
-    def __sub__(self: Self, other: Iterable[_T]) -> Self: ...
-    def difference_update(self, other: Iterable[_T]) -> None: ...
-    def __isub__(self: Self, other: Iterable[_T]) -> Self: ...
+    def __sub__(self: Self, other: Iterable[Any]) -> Self: ...
```

Comment:
Why did you change this (as well as `__and__`)? If nothing else, it seems a bit off-topic for this PR, but also inconsistent with the super class annotations.